esdt-safe/src/to_sovereign/create_tx.rs

```diff
@@ -79,13 +78,23 @@ pub trait CreateTxModule:
                     payment.token_nonce,
                     current_token_data.clone(),
                 ));
-
                 event_payments.push(event_payment);
             } else {
                 let sov_token_id = self
                     .multiversx_to_sovereign_token_id(&payment.token_identifier)
                     .get();
 
+                if !sov_token_id.is_valid_esdt_identifier() {
```

Comment:
`sov_token_id` will have a prefix. Are you sure `is_valid_esdt_identifier` will return `true` for a token with prefix? Otherwise just use `sov_token_id NOT EMPTY` somehow